pom.xml

```diff
@@ -134,6 +134,17 @@
       <plugin>
         <artifactId>maven-failsafe-plugin</artifactId>
       </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-jar-plugin</artifactId>
+        <configuration>
+          <archive>
+            <manifestEntries>
+              <Automatic-Module-Name>com.spotify.completablefutures</Automatic-Module-Name>
```

Comment:
Knowing nothing about JPMS: The package name in use is `com.spotify.futures`. Should they match?

Reply:
It seems like that is [the convention](https://blog.joda.org/2017/04/java-se-9-jpms-module-naming.html) but not required, but I figured we'd have a conflict if we ever did the same thing in https://github.com/spotify/futures-extra which has the same package name.

Reply:
Yes, in futures-extra we have this: https://github.com/spotify/futures-extra/blob/master/src/main/java/module-info.java#L17

Reply:
oh nice, I did not realize there was a proper module-info there already